blueprintapi/settings.py

```diff
@@ -22,11 +23,21 @@
     "handlers": {
         "console": {
             "class": "logging.StreamHandler",
+            "stream": sys.stdout,
+            "formatter": "simple",
         },
     },
-    "root": {
-        "handlers": ["console"],
-        "level": "WARNING",
+    "formatters": {
+        "simple": {
+            "format": "[{asctime}] [{levelname}] {message}",
+            "style": "{",
+        },
+    },
+    "loggers": {
+        "": {
```

Comment:
Why not add a name for the logger here?

Reply:
If I give the formatter a name the format defined in `simple` doesn't apply to all of the loggers, but only to ones called by the name or that are using the `simple` formatter.

Reply:
I see. In that case, why not apply it to `root`?